app/keepers.go

```diff
@@ -0,0 +1,326 @@
+		app.AccountKeeper, app.BankKeeper, app.ScopedTransferKeeper,
+	)
+	app.TransferKeeper = &transferKeeper
+	app.transferModule = transfer.NewAppModule(*app.TransferKeeper)
+
+	// Create static IBC router, add transfer route, then set and seal it
+	ibcRouter := porttypes.NewRouter()
+	ibcRouter.AddRoute(ibctransfertypes.ModuleName, app.transferModule)
+	app.IBCKeeper.SetRouter(ibcRouter)
+
+	app.Bech32IBCKeeper = bech32ibckeeper.NewKeeper(
+		app.IBCKeeper.ChannelKeeper, appCodec, keys[bech32ibctypes.StoreKey],
+		app.TransferKeeper,
+	)
+
+	// TODO: Should we be passing this instead of bank in many places?
+	// Where do we want send coins to be cross-chain?
+	app.Bech32ICS20Keeper = bech32ics20keeper.NewKeeper(
+		app.IBCKeeper.ChannelKeeper,
+		app.BankKeeper, app.TransferKeeper,
+		app.Bech32IBCKeeper,
+		app.TransferKeeper,
+		appCodec,
+	)
+
+	// create evidence keeper with router
+	// If evidence needs to be handled for the app, set routes in router here and seal
+	app.EvidenceKeeper = evidencekeeper.NewKeeper(
+		appCodec, keys[evidencetypes.StoreKey], app.StakingKeeper, app.SlashingKeeper,
+	)
+
+	app.ClaimKeeper = claimkeeper.NewKeeper(
+		appCodec,
+		keys[claimtypes.StoreKey],
+		app.AccountKeeper,
+		app.BankKeeper, app.StakingKeeper, app.DistrKeeper)
+
+	gammKeeper := gammkeeper.NewKeeper(
+		appCodec, keys[gammtypes.StoreKey],
+		app.GetSubspace(gammtypes.ModuleName),
+		app.AccountKeeper, app.BankKeeper, app.DistrKeeper)
+	app.GAMMKeeper = &gammKeeper
+
+	app.LockupKeeper = lockupkeeper.NewKeeper(
+		appCodec, keys[lockuptypes.StoreKey],
+		// TODO: Visit why this needs to be deref'd
+		*app.AccountKeeper,
+		app.BankKeeper)
+
+	app.EpochsKeeper = epochskeeper.NewKeeper(appCodec, keys[epochstypes.StoreKey])
+
+	app.IncentivesKeeper = incentiveskeeper.NewKeeper(
+		appCodec, keys[incentivestypes.StoreKey],
+		app.GetSubspace(incentivestypes.ModuleName),
+		*app.AccountKeeper,
+		app.BankKeeper, app.LockupKeeper, app.EpochsKeeper)
+
+	mintKeeper := mintkeeper.NewKeeper(
+		appCodec, keys[minttypes.StoreKey],
+		app.GetSubspace(minttypes.ModuleName),
+		app.AccountKeeper, app.BankKeeper, app.DistrKeeper, app.EpochsKeeper,
+		authtypes.FeeCollectorName,
+	)
+	app.MintKeeper = &mintKeeper
+
+	poolIncentivesKeeper := poolincentiveskeeper.NewKeeper(
+		appCodec,
+		keys[poolincentivestypes.StoreKey],
+		app.GetSubspace(poolincentivestypes.ModuleName),
+		app.AccountKeeper,
+		app.BankKeeper,
+		app.IncentivesKeeper,
+		app.DistrKeeper,
+		distrtypes.ModuleName,
+		authtypes.FeeCollectorName,
+	)
+	app.PoolIncentivesKeeper = &poolIncentivesKeeper
+
+	txFeesKeeper := txfeeskeeper.NewKeeper(
+		appCodec,
+		keys[txfeestypes.StoreKey],
+		app.GAMMKeeper,
+	)
+	app.TxFeesKeeper = &txFeesKeeper
+
+	// register the proposal types
+	// TODO: This appears to be missing tx fees proposal type
+	govRouter := govtypes.NewRouter()
+	govRouter.AddRoute(govtypes.RouterKey, govtypes.ProposalHandler).
```

Comment:
Gov router is missing the proposal type for TxFees, this needs an issue to get added in next release.

Reply:
New issue made